app/routes/application.js

```diff
@@ -25,7 +25,7 @@ export default Route.extend({
     }
 
     const page = this.url;
-    const title = this.getWithDefault('currentRouteName', 'unknown');
+    const title = this.currentRouteName || 'unknown';
```

Comment:
can you please make use of `??` instead of `||`. I know in this case the currentRouteName is supposed to be a string so it will only be falsey if it is empty, but I would prefer to make use of `??` for default values as a standard 👍

Reply:
Thank you for reminding me[ Coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) use. Almost forgot :)